### PR TITLE
Added styling rules to our linter; these included comments spacing an…

### PR DESCRIPTION
**Pull request by rainrivas**
…d caps, camelcase for vars (ALL CAPS snake_case excluded) and configured comma style/spacing so we can be consistent there as well.

Also added test files so we can see what's happening.

### CODE REVIEW

**Review by AlChiu — APPROVED** [review]
Just tested it myself. Looking good on my end. It is catching all of the double quotes.

**Review by jwu910 — DISMISSED** [review]

**Review by phuchle — COMMENTED** [review]
Few things I noticed on my machine (Atom running on Ubuntu) for lintConfigTest:

- Line 18 comment should error but doesn't
- Lines 90..92 should error out I believe.

**Review by psyduc — COMMENTED** [review]
what was the change to line 19?

nvm, i see the comma

**Review by psyduc — APPROVED** [review]

**Comment by rainrivas** [comment]
@phuchle 
>Few things I noticed on my machine (Atom running on Ubuntu) for lintConfigTest:
>
>Line 18 comment should error but doesn't
>Lines 90..92 should error out I believe.

do you want me to work on this and fix it for atom? What config do I need to set up the atom linter (I have atom on one of these machines)
**EDIT** Regarding your ln90-92 comment; those should not error as stated in the ESLint source https://eslint.org/docs/rules/camelcase
This is because those lines are fucntion calls which are not flagged
>This rule only flags definitions and assignments but not function calls

**Comment by phuchle** [comment]
@rainrivas 
Yeah if you could that would be great.  Packages I have installed on Atom for linting: linter, linter-ui-default, linter-eslint.

**Comment by rainrivas** [comment]
@phuchle I only see atom missing 2 errors my sublime caught and they are tabbing vs space (i think atom auto corrects tabs to spaces). So I don't see a problem there. Also you mentioned that line 18 should error but ln18 is blank. Not sure what you are referring to, but it's a comment block apparently
**EDIT**@phuchle OOOOHHHH ln18 in `test/` !! That's actually a typo on my part and that shouldn't be an error because it's a second line comment which is handled here:
`"ignoreConsecutiveComments": true` (under the capitalize comments `.eslintrc.json` ln22-30

File: test/lintConfigTest.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+var a, b, c = 0; // This should error
+
+let d,
+    e;
+
+let f,
+    g = 0;
+
+//this is a bad comment, (no space, no cap)
+// this is not a bad (despite no caps) because it's a consecutive comment
```
Comment (rainrivas) [inline test/lintConfigTest.js:18]: @phuchle i fixed the expected behaviour. Look at ln 21-26 for the error that was expected previously

**Comment by rainrivas** [comment]
Merging and closing this PR so we can do final review on Duc's PR (PR#6)

**Comment by jwu910** [comment]
SWEET

**Pull request merged**